src/tiles.jl

```diff
@@ -122,7 +122,7 @@ function TileGrid(bbox::Extent, zoom::Int, crs::WGS84)
     # Mercantile splits the bbox in two along the antimeridian if this happens.
     # Decide if that case should be handled here or before, also considering
     # antimeridian discussion in https://github.com/rafaqz/Extents.jl/issues/4
-    @assert bbox.X[1] < bbox.X[2]
+    @assert bbox.X[1] <= bbox.X[2]
```

Comment:
```suggestion
    if bbox.X[1] > bbox.X[2]
        throw(ArgumentError("TileGrid Extent has xmin > xmax"))
    end
```

Reply:
Ah sorry I missed this comment

Reply:
Ha I was too late. Though in a way it is unrelated.